libmscore/repeatlist.cpp

```diff
@@ -260,19 +284,49 @@ void RepeatList::unwind()
             return;
 
 // qDebug("unwind===================");
+
+      for (Measure* m = fm; m; m = m->nextMeasure())
+            m->setPlaybackCount(0);
+
+      // partition score by section breaks, and unwind individual sections seperately
```

Comment:
I also notice that section break on a frame will not reset the starting measure of next section to 1